dbms/src/Interpreters/Join.h

```diff
@@ -94,19 +99,28 @@ class Join
          const String & req_id,
          bool enable_fine_grained_shuffle_,
          size_t fine_grained_shuffle_count_,
+         size_t max_bytes_before_external_join_,
+         const SpillConfig & build_spill_config_,
+         const SpillConfig & probe_spill_config_,
+         Int64 join_restore_concurrency_,
```

Comment:
I prefer to remove `join_restore_concurrency` and then the logic of join restoring can be simplified.

Reply:
> I prefer to remove `join_restore_concurrency` and then the logic of join restoring can be simplified.

it has already discussed in meeting.